samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java

```diff
@@ -1,2 +1,173 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
\ No newline at end of file
+// Licensed under the MIT License.
+package com.microsoft.bot.sample.multilingual;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.bot.builder.ActivityHandler;
+import com.microsoft.bot.builder.UserState;
+import com.microsoft.bot.builder.StatePropertyAccessor;
+import com.microsoft.bot.builder.TurnContext;
+import com.microsoft.bot.builder.MessageFactory;
+import com.microsoft.bot.schema.ChannelAccount;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.CardAction;
+import com.microsoft.bot.schema.ActionTypes;
+import com.microsoft.bot.schema.SuggestedActions;
+import com.microsoft.bot.schema.Attachment;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * This bot demonstrates how to use Microsoft Translator.
+ * More information can be found
+ * here https://docs.microsoft.com/en-us/azure/cognitive-services/translator/translator-info-overview.
+ */
+public class MultiLingualBot extends ActivityHandler {
+    private static final StringBuilder WELCOME_TEXT =
+        new StringBuilder("This bot will introduce you to translation middleware. ")
+        .append("Say 'hi' to get started.");
+    private static final String ENGLISH_ENGLISH = "en";
+    private static final String ENGLISH_SPANISH = "es";
+    private static final String SPANISH_ENGLISH = "in";
+    private static final String SPANISH_SPANISH = "it";
```

Comment:
Why all these `static`? In [C#](https://github.com/microsoft/BotBuilder-Samples/blob/main/samples/csharp_dotnetcore/17.multilingual-bot/Bots/MultiLingualBot.cs#L25) they are not `static`

Reply:
I followed this examples in stackoverflow
- [Example 1](https://stackoverflow.com/a/66076/6469915)
- [Example 2](https://stackoverflow.com/a/12519516/6469915)

But we can remove the static anyway

Reply:
Well, remove the static from the variables throws an error that cannot be use them in static methods, so, as the variables are use them in static methods we need to leave them as static

Reply:
Ack